src/modules/core/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import ReactSlider from 'rc-slider';
+import 'rc-slider/assets/index.css';
+
+type Appearance = {
+  theme?: 'primary' | 'danger';
+};
+
+interface Props {
+  value: number;
+  max?: number;
+  limit?: number;
+  appearance?: Appearance;
+  onChange: (val: any) => void;
+}
+
+const displayName = 'Slider';
+
+const Slider = ({ value, max = 100, onChange, limit, appearance }: Props) => {
+  const [sliderValue, setSliderValue] = useState<number>(value);
+
+  const gradientStopPercentage = useMemo(() => {
+    return limit ? Math.round((limit / max) * 100) : 0;
+  }, [limit, max]);
+
+  const onSliderChange = useCallback(
+    (val): void => {
+      if ((limit && sliderValue < limit) || val < sliderValue || !limit) {
+        setSliderValue(val);
+        onChange(val);
+      }
+      if (limit && sliderValue > limit) {
+        setSliderValue(limit);
+        onChange(limit);
+      }
+    },
+    [setSliderValue, onChange, limit, sliderValue],
+  );
```

Comment:
Using the `setValue` method imported from formik, you'll need to call it every time the value changes:

```suggestion
  const onSliderChange = useCallback(
    (val): void => {
      if ((limit && sliderValue < limit) || val < sliderValue || !limit) {
        setSliderValue(val);
        setValue(val);
        onChange(val);
      }
      if (limit && sliderValue > limit) {
        setSliderValue(limit);
        setValue(limit);
        onChange(limit);
      }
    },
    [setSliderValue, onChange, limit, sliderValue],
  );
```

And that's it, now this will function as form element.

If you really want to go a step further, you can also add a `isConnected` prop (that defaults to `true`) and use that only make this a connected form element if that is set. 

But that is optional, as most likely we'll only use this in a form configuration.